plugins/vswitch/vswitch.cpp

```diff
@@ -609,6 +611,67 @@ class wf_vswitch_global_plugin_t : public wf::per_output_plugin_t<vswitch>
 
         return wf::ipc::json_ok();
     };
+
+    wf::ipc::method_callback send_view = [=] (const wf::json_t& data)
+    {
+        uint64_t x = wf::ipc::json_get_uint64(data, "x");
+        uint64_t y = wf::ipc::json_get_uint64(data, "y");
+        uint64_t output_id = wf::ipc::json_get_uint64(data, "output-id");
+        std::optional<uint64_t> view_id = wf::ipc::json_get_optional_uint64(data, "view-id");
```

Comment:
I am a bit confused, if the method is `send-view` then why is the view optional? IPC clients can call `set-workspace` if they want to switch without a view ...